src/main/kotlin/com/jay/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,25 @@
+package com.jay.calculator
+
+import java.lang.IllegalArgumentException
+
+object StringCalculator {
+    fun calculate(input: String?): Int {
+        validate(input)
+        val split = input!!.split(" ")
+        var result: Int = toInt(split[0])
```

Comment:
```suggestion
        var result: Int = toInt(split.first())
```
0번째 값을 사용해도 좋지만, first() 함수를 사용하시면 좀 더 코틀린스럽게 코드를 작성하실 수 있어요. 😄

Reply:
덕분에 collections더 살펴보게되었습니다 감사합니다!